fixed_frame:
object_frame:

reach_object:
  object_mesh_filename:
  pcd_file:
  n_neighbors:
  cloud_output_res:

move_group:
  kinematics:
  manipulability:

constraints:
  get_avg_neighbor_count:
  cost_function:
  distance_threshold:
  compare_dbs:
